packages/manager/apps/pci-gateway/src/pages/add/NetworkStep.tsx

```diff
@@ -168,11 +175,28 @@ export const NetworkStep = (): JSX.Element => {
       });
     },
     onError: (error) => {
-      addError(
-        t('pci_projects_project_public_gateways_add_error', {
-          message: error,
-        }),
-      );
+      const errorMessage =
+        isApiCustomError(error) && isMaxQuotaReachedError(error) ? (
+          <Trans
+            ns="add"
+            i18nKey="pci_projects_project_public_gateways_quota_error"
+            components={{
+              Link: (
+                <GuideLink
+                  href={`${hrefProject}/quota`}
+                  isTargetBlank={false}
+                />
+              ),
+            }}
+          />
+        ) : (
+          t('pci_projects_project_public_gateways_add_error', {
+            message: isApiCustomError(error) ? error.response.data.message : '',
```

Comment:
if error is not an apiCustomError, the message will be:
"Une erreur est survenue lors de la création de la Gateway: ", I think it could be nice to end message without `:` in this case, what do you think?